disabled-packages/ldc/build.sh

```diff
@@ -34,7 +34,7 @@ termux_step_post_extract_package () {
 		$TERMUX_PKG_CACHEDIR/dmd.${DMD_COMPILER_VERSION}.linux.tar.xz \
 		e48783bd91d77bfdcd702bd268c5ac5d322975dd4b3ad68831babd74509d2ce9
 
-	sed "s#\@TERMUX_C_COMPILER\@#$TERMUX_STANDALONE_TOOLCHAIN/bin/$TERMUX_HOST_PLATFORM-clang#" \
+	sed "s#\@TERMUX_C_COMPILER\@#$TERMUX_TOPDIR/_lib/$TERMUX_NDK_VERSION-$TERMUX_ARCH-$TERMUX_PKG_API_LEVEL-v8/bin/$TERMUX_HOST_PLATFORM-clang#" \
```

Comment:
Sure! I've moved up the declaration in https://github.com/termux/termux-packages/commit/28c3a74ab38f6528baf41436fcdfb5f75788991a, so this can be changed back to TERMUX_STANDALONE_TOOLCHAIN.